hack/dind-cluster.sh

```diff
@@ -631,7 +669,19 @@ case "${1:-""}" in
     fi
 
     NETWORK_PLUGIN="$(get-network-plugin "${NETWORK_PLUGIN}")"
-    start "${OS_ROOT}" "${CONFIG_ROOT}" "${DEPLOYED_CONFIG_ROOT}" \
+
+    # OVN requires CNI network plugin and OVN_ROOT to be set
+    if [[ "${NETWORK_PLUGIN}" = "ovn" ]]; then
+      NETWORK_PLUGIN="cni"
+      if [[ -z "${OVN_ROOT}" ]]; then
```

Comment:
If it can be unset use `:-` defaulting

Reply:
@stevekuznetsov I defaulted it above:

```
ADDITIONAL_ARGS=""

OVN_ROOT="${OVN_ROOT:-}"

case "${1:-""}" in
  start)
    BUILD=
```

is that sufficent?

Reply:
Yes, sorry -- didn't catch that